admin-dev/themes/new-theme/js/components/grid/extension/bulk-open-tabs-extension.js

```diff
@@ -0,0 +1,85 @@
+import Router from '../../router';
```

Comment:
Move this after the license block please

Reply:
Oooohhh so this is where it was hidden!! Damn PHPStorm I didn't understand where the import came from ^^